chef/cookbooks/metasploitable/recipes/flags.rb

```diff
@@ -0,0 +1,116 @@
+#
+# Cookbook:: metasploitable
+# Recipe:: flags
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+include_recipe 'metasploitable::iis'
+include_recipe 'metasploitable::wamp'
+
+cookbook_file 'C:\inetpub\wwwroot\six_of_diamonds.zip' do
+  source 'flags/six_of_diamonds.zip'
+  action :create
+end
+
+cookbook_file 'C:\WINDOWS\System32\jack_of_clubs.png' do
+  source 'flags/jack_of_clubs.png'
+  action :create
+end
+
+batch 'Jack of Clubs' do
+  code <<-EOH
+    attrib +h "C:\\WINDOWS\\System32\\jack_of_clubs.png"
+    cacls "C:\\WINDOWS\\System32\\jack_of_clubs.png" /t /e /g SYSTEM:f
+    cacls "C:\\WINDOWS\\System32\\jack_of_clubs.png" /R Administrators /E
+    cacls "C:\\WINDOWS\\System32\\jack_of_clubs.png" /R Users /E
+  EOH
+end
+
+cookbook_file 'C:\Windows\three_of_spades.png' do
+  source 'flags/three_of_spades.png'
+  action :create
```

Comment:
Should you use the `mode`, `owner`, `group` etc.. attributes instead of the `batch` resource below?